daemon/mgr/container.go

```diff
@@ -247,7 +245,7 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		}
 	}
 	if mgr.NameToID.Get(name).Exist() {
-		return nil, fmt.Errorf("container with name %s already exist", name)
+		return nil, errors.Wrap(ErrAlreadyExisted, fmt.Sprintf("container with name %s already exist", name))
```

Comment:
The error message shows like `container with name abcd already exist: already existed`.
How about simplifying to be `container abcd: already existed`?